src/React/Components/Home/home.css

```diff
@@ -0,0 +1,9 @@
+.Home .lander {
```

Comment:
I'd prefer css to be in the files using them, only global css would be in a file! Look into libraries for doing CSS within our .js react files.

Reply:
I updated the PR and remove the CSS files